### PR TITLE
Fix test for container events

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

File: integration-cli/docker_cli_events_test.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func (s *DockerSuite) TestEventsContainerEvents(c *check.C) {
 	if createEvent[len(createEvent)-1] != "create" {
 		c.Fatalf("event should be create, not %#v", createEvent)
 	}
-	if attachEvent[len(createEvent)-1] != "attach" {
+	if attachEvent[len(attachEvent)-1] != "attach" {
```
Comment (runcom) [inline integration-cli/docker_cli_events_test.go]: how come this test was passing till now w/o this fixed?

Comment (HuKeping) [inline integration-cli/docker_cli_events_test.go]: They are very similar and we only care about the last filed of each record here
```
2015-09-24T19:53:13.317643629+08:00 0e610126cfa5e279b28ffb208698bb1a9b0e824cb449f00ad6dd2597c090af2d: (from busybox) create
2015-09-24T19:53:13.360017014+08:00 0e610126cfa5e279b28ffb208698bb1a9b0e824cb449f00ad6dd2597c090af2d: (from busybox) attach
2015-09-24T19:53:13.365965186+08:00 0e610126cfa5e279b28ffb208698bb1a9b0e824cb449f00ad6dd2597c090af2d: (from busybox) start
2015-09-24T19:53:13.484393306+08:00 0e610126cfa5e279b28ffb208698bb1a9b0e824cb449f00ad6dd2597c090af2d: (from busybox) die
2015-09-24T19:53:13.770341115+08:00 0e610126cfa5e279b28ffb208698bb1a9b0e824cb449f00ad6dd2597c090af2d: (from busybox) destroy
```

**Comment by runcom** [comment]
LGTM ping @duglin

**Comment by duglin** [comment]
hmmm odd that this ever worked... let me check something first....

**Comment by duglin** [comment]
ok - I see. It passes w/o this fix because the 'create' and 'attach' events look the same w.r.t. the number and order of the 'fields' - so using the wrong event to grab the data was safe.  But this fix is still a good one.
LGTM

**Pull request merged**